### ISSUE TITLE
0.74.1 crashloop when uploading big file to channel (70MB)

### ISSUE THREAD

**Issue by himpierre** [RocketChat/Rocket.Chat#13355]
### Description:

I was trying to upload a mp4 to a channel. During that process Rocketchat crashloops. It does not recover even after a restart of rocketchat. Had to restore from backup. Error message:

```
<--- Last few GCs --->

[23045:0x559902607170]    45650 ms: Mark-sweep 882.2 (969.4) -> 882.2 (969.4) MB, 260.7 / 0.0 ms  allocation failure GC in old
 space requested
[23045:0x559902607170]    45938 ms: Mark-sweep 882.2 (969.4) -> 882.2 (919.4) MB, 287.2 / 0.0 ms  last resort GC in old space 
requested
[23045:0x559902607170]    46229 ms: Mark-sweep 882.2 (919.4) -> 882.2 (912.9) MB, 290.8 / 0.0 ms  last resort GC in old space 
requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x3e0a51da1769 <JSObject>
    1: /* anonymous */ [/opt/rocketchat-0.74.1/programs/server/packages/base64.js:~61] [pc=0x7f1cd34163e](this=0xaef9b40c2e9 <
JSGlobal Object>,elm=49,i=56406920)
    2: arguments adaptor frame: 3->2
    3: forEach(this=0xca90225e9b9 <Uint8Array map = 0x25db677831d1>)
    4: encode [/opt/rocketchat-0.74.1/programs/server/packages/base64.js:61] [bytecode=0x3817bee5dc51 offset=144](this=0x1d966
1aa2...
```

### Steps to reproduce:

1. Upload a big file (70MB+) to a channel
2. Rocketchat crashes
3. Rocketchat restart does not help

### Expected behavior:

Upload succeeds or a info message appears (file to big or somethin)

### Actual behavior:

Rocketchat crashloops.

### Server Setup Information:

- Version of Rocket.Chat Server: 0.74.1
- Operating System: Linux
- Deployment Method: tar
- Number of Running Instances: 1
- DB Replicaset Oplog: no
- NodeJS Version: 8.15.0
- MongoDB Version: 3.6.10

**Comment by reetp** [comment]
Server spec, particularly RAM?

**Comment by himpierre** [comment]
Rocketchat runs on virtual machine with 4GB of RAM. It does not seem the VM was running out of RAM. Is that what the error message says?

thanks and cheers
t.

**Comment by moonwolf-github** [comment]
Looks similar (related maybe?) to my #13191.

**Comment by himpierre** [comment]
Yes it does. WebDAV integrations is enabled on my installation but not used in this case. I just hit the + and "Computer" to upload the file.

**Comment by himpierre** [comment]
Should i try to upload that file to https://open.rocket.chat?

**Comment by reetp** [comment]
They may have file limits on there.

See if a dev comes back to you here first.

**Comment by KevLehman** [comment]
This pull request has been inactive for a while, so we’re closing it to keep the backlog manageable.
If you’d like to continue working on it, feel free to reopen or submit a new PR. Thanks for the contribution!
